test/provider/expect.unittest.ts

```diff
@@ -308,6 +308,16 @@ describe('When creating an external connection', () => {
 function expectTest(testWindow: TestWindow, appDirectoryResultTime: number, resultParams: ResultParam[]): void {
     const testParams = buildTestParams(testWindow, resultParams);
 
+    function getEntityType(entityIdentity: Identity, testWindowIdentity: Identity): EntityType {
```

Comment:
This feels a bit round-a-bout

Would do:

- Put `const testWindowEntityType = testWindow.appType === 'non-directory-external' ?  EntityType.EXTERNAL_CONNECTION : return EntityType.WINDOW` around line 324.

- Use that const in the `emit`s

- Put the extracted function back in-line, and use that const